.gitignore

```
@@ -13,3 +13,5 @@
 
 *~
 /example[0-9]+
+
+vendor/
```

```suggestion
/vendor/
```

Will ensure this matches at the top level, not any folder named "vendor"